install/helm/agones/values.yaml

```diff
@@ -134,6 +138,10 @@ agones:
     numWorkers: 100
     apiServerQPS: 400
     apiServerQPSBurst: 500
+    maxCreationParalellismFlag: 16
+    maxGameServerCreationsPerBatch: 64
+    maxDeletionParallelismFlag: 64
+    maxGameServerDeletionsPerBatch: 64
```

Comment:
These are not needed since this blob is for extension

Reply:
Oh that's true, just noticed that, thanks!